charts/merlin/values.yaml

```diff
@@ -26,167 +26,241 @@ deployment:
   #     value: master
   #     effect: NoSchedule
   tolerations: []
-environment: dev
-deploymentLabelPrefix: "gojek.com/"
-pyfuncGRPCOptions: "{}"
-loggerDestinationURL: "http://yourDestinationLogger"
-queue:
-  numOfWorkers: 1
-mlpApi:
-  apiHost: http://mlp.mlp:8080
-feastCoreApi:
-  apiHost: http://feast-core.mlp:8080/v1
+
+  # -- List of string containing additional Merlin API server arguments. For
+  # example, multiple "-config" can be specified to use multiple config files
+  extraArgs: []
```

Comment:
Added these `extra-` blocks because they seem to be potentially useful for any quick form of iteration during development.